lib/Tvm.js

```diff
@@ -311,16 +310,16 @@ class Tvm {
       try {
         await this._validateOWCreds(params.owApihost, params.owNamespace, params.owAuth)
         await this._validateNamespaceInApprovedList(params.owNamespace, params.approvedList)
-        this.track(params, 'request_count')
+        this.track(params.owNamespace, 'request_count')
       } catch (e) {
         if (!e.code || !KNOWN_4XX_ERRORS.includes(e.code)) {
           logger.warn(`server error: ${e.message} - request Id - ${requestId}`)
           // return 500 error if its not a known 4xx error or no e.code set
-          this.track(params, 'error_count', '500')
+          this.track(params.owNamespace, 'error_count', '500')
           return Tvm._userErrorResponse('server error', 500)
         }
         logger.warn(`unauthorized request: ${e.message} - request Id - ${requestId}`)
-        this.track(params, 'user_error_count', '403')
+        this.track(params.owNamespace, 'user_error_count', '403')
```

Comment:
This just needs to be  `this.track(params.owNamespace, 'error_count', '403')`

Reply:
After discussing with Sandeep, I am working on supporting user_error_count in devx metrics and lib